.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,15 +1,18 @@
-This pull request addresses issue #...
-
 <!--- Please read each of the following items and confirm by replacing
  !--the [ ] with a [X] --->
 
 - [ ] I hereby agree to dual licence this and any previous contributions under both
 the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.
 
-- [ ] I have read the ``CONTRIBUTING.rst`` file, have run ``flake8`` locally, and
-understand that AppVeyor and TravisCI will be used to confirm the Biopython unit
+- [ ] I have read the ``CONTRIBUTING.rst`` file, have run ``pre-commit`` locally,
+and understand that AppVeyor and TravisCI will be used to confirm the Biopython unit
 tests and style checks pass with these changes.
 
 - [ ] I have added my name to the alphabetical contributors listings in the files
 ``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
 already, or do not wish to be listed. (*This acknowledgement is optional.*)
+
+<!--- Briefly describe the changes included in this pull request below
+ !--- starting with the words 'Closes issue #... --->
```

Comment:
How about:

``starting with 'Closes issue #...' if appropriate``

Reply:
Sounds good!

Reply:
Got it!